pages/donate/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box, Center, Container, Flex, Link, Text } from '@chakra-ui/react';
+import NextLink from 'next/link';
+
+const Donate: React.FC = () => {
+  return (
+    <>
+      <Flex gap={'150px'} flexDir={'column'}>
```

Comment:
We should probably have `mt={'50px'}` or something like that because the text is way too close to the top and it will be uncomfortably close to the navbar once it gets added. Also the Figma page has some whitespace between the navbar and text so please follow that.